src/rules/value-list-max-empty-lines/README.md

```diff
@@ -0,0 +1,108 @@
+# value-list-max-empty-lines
+
+Limit the number of adjacent empty lines within value lists.
+
+```css
```

Comment:
```css
a {
  box-shadow:
    inset 0 2px 0 #dcffa6,
                    /* ← */
    0 2px 5px #000; /* ↑ */
}                   /* ↑ */
/**                    ↑
 *       This empty line */
```

The default `indentation` setting for values is one level more than the declarations. This updates the example to match that.